app/frontend/src/features/messages/messagelist/InfiniteMessageLoader.tsx

```diff
@@ -82,44 +82,24 @@ export default function InfiniteMessageLoader({
   }, []);
 
   //**  Keep place or keep at bottom on window resize (i.e. keyboard popup on mobile (hopefully)) **//
-  // Save the current height outside of the eventlistener
-  const [currentClientHeight, setCurrentClientHeight] = useState(0);
-  useLayoutEffect(() => {
-    scrollRef.current && setCurrentClientHeight(scrollRef.current.clientHeight);
-  });
-
-  useLayoutEffect(() => {
+  useEffect(() => {
     const updateMessagePosition = () => {
       if (!scrollRef.current) return;
-      const currentlyAtBottom =
-        scrollRef.current.scrollHeight <=
-        scrollRef.current.scrollTop + currentClientHeight;
-
-      if (currentlyAtBottom) {
-        // Send to bottom on window resize
-        scrollRef.current.scroll(0, scrollRef.current.scrollHeight);
-      }
-      setCurrentClientHeight(scrollRef.current.clientHeight);
+      scrollRef.current.scroll(0, scrollRef.current.scrollHeight);
     };
-
-    if (!scrollRef.current) return;
     window.addEventListener("resize", updateMessagePosition);
     return () => window.removeEventListener("resize", updateMessagePosition);
-  }, [currentClientHeight]);
+  }, []);
 
   //** Scroll to the bottom on new message  **//
-  // Saving messages count to know whether a new message has been created
-  const { messageId } = children.props.messages[0];
-  const [newestMessageID, setNewestMessageID] = useState(messageId);
-
+  const [savedMessageId, setSavedMessageId] = useState(latestMessageId);
   useEffect(() => {
```

Comment:
Needs to be `useLayoutEffect` if you're scrolling or manipulating the DOM.

Reply:
The other one doesn't need it because  the scrolling happens in a callback, not in the effect.

Reply:
> not if someone else just sent us a message

Yes! Bloody hell, this infuriates me when I'm trying to read something in a group chat and new messages keep pushing me to the bottom. Good catch